doc/cheatsheet.md

```diff
@@ -0,0 +1,255 @@
+% djot cheatsheet
```

Comment:
Use a level-1 heading for this?  This is pandoc syntax and won't work on gfm.

Reply:
Oh. Right. Habit from working with Pandoc. Thanks.